tests/test_cli.py

```diff
@@ -58,3 +58,19 @@ def test_cmd_server_with_cloud(self, mock_state_loader_class, mock_run, mock_get
         self.runner.invoke(cli_command_server, ['--cloud', '--password', 'unittest', '--cloud-token', 'unittest'])
         mock_state_loader_class.assert_called_once()
         mock_run.assert_called_once()
+
+
+class TestCommandRun(TestCase):
+    def setUp(self):
+        self.runner = CliRunner()
+        pass
+
+    @patch.object(RecceContext, 'verify_required_artifacts')
+    @patch('recce.cli.asyncio.run')
+    def test_cmd_run(self, mock_asyncio_run, mock_verify_required_artifacts):
+        from recce.cli import run
+
+        mock_verify_required_artifacts.return_value = True, None
+
+        self.runner.invoke(run, [])
+        mock_asyncio_run.assert_called_once()
```

Comment:
Hi Kent,
I found that mocking `asyncio.run` could lead to the warning message:
`sys:1: RuntimeWarning: coroutine 'cli_run' was never awaited`

should we change to `@patch('recce.cli.cli_run')` to check called once?

Reply:
Yes! You are right!
Changed to mock `cli_run` instead of mocking `asyncio.run`

Reply:
Yes! You are right!
Changed to mock cli_run instead of mocking asyncio.run